layouts/shortcodes/event-card.html

```diff
@@ -0,0 +1,132 @@
+{{ $audio_recording_id := .Get "audioRecordingId" }}
+{{ $audio_event_id := .Get "audioEventId" }}
+{{ $label := .Get "label" }}
+
+{{- if not $audio_recording_id -}}
+    {{ errorf "Missing audioRecordingId for event card with label %s" $label }}
+{{- end -}}
+
+{{- if not $audio_event_id -}}
+    {{ errorf "Missing audioEventId for event card with label %s" $label }}
+{{- end -}}
+
+{{ $id := printf "event-card-%03d" $.Ordinal }}
+
+<div id="{{ $id }}" class="card-host">
+    <sl-card class="spectrogram-card">
+        <div class="card-header" slot="header">
+            <h3 class="event-label">{{ $label }}</h3>
+            <oe-media-controls for="{{ $id }}-spectrogram"></oe-media-controls>
+        </div>
+
+        <div class="card-body">
+            <oe-annotate id="{{ $id }}-annotations">
+                <oe-axes>
+                    <oe-indicator>
+                        <oe-spectrogram
+                            id="{{ $id }}-spectrogram"
+                            class="event-spectrogram"
+                        ></oe-spectrogram>
+                    </oe-indicator>
+                </oe-axes>
+            </oe-annotate>
+        </div>
+
+        {{ .Inner }}
+    </sl-card>
+</div>
+
+<script type="module">
+    const spectrogramTarget = document.getElementById("{{ $id }}-spectrogram");
+    const annotationTarget = document.getElementById("{{ $id }}-annotations");
+
+    // TODO: Refactor this function once we support object annotations
```

Comment:
I actually don't mind not having an object model in this case.

Setting attributes is totally fine, especially when we have only a few instances, and updates are slow (in this case, never happen)